EIPS/eip-7539.md

```diff
@@ -0,0 +1,101 @@
+
+There should be a framework for handling extensions both retroactively and in the future to improve ERC accessibility. Without this, ERC extensions may be introduced which break compatibility and composability in nontrivial ways. Entire classes of issues can be removed by having a strong definition and standard for what an Extension can and cannot change. 
+
+Extensions also allow the evolution and discussion of standards to adopt a clear graphical structure demonstrating where functionality branches and extends from lower level standards.
+
+Having a conventions for how to properly specify the critical changes in functionality improves the clarity of writing, editing, and implementing ERCs in the future.
+
+
+## Specification
+
+### Definitions
+
+Usage of the term Extension below is shorthand for an ERC Extension.
+
+An Extension is any ERC which has all of the following properties:
+
+An Extension MUST implement at least one existing ERC and add additional functionality and specificity to that ERC.
+
+An Extension MUST be fully compatible at the function interface level. 
+	I.e. all functions in the extended interface MUST implement the exact same function signature and selector.
+
+	An Extension MAY NOT have the same parameter names or state mutability.
+
+An Extension MAY include some non-compliance at the implementation level specification, as long as the overriden behavior is fully specified.
+	
+	Specifically, mutable functions MUST remain mutable and immutable functions MUST remain immutable. I.e. the state mutability between `payable` and `non-payable` MAY interchange and the state mutablity between `view` and `pure` MAY interchange. 
+
+### Extension Naming
+
+An Extension SHOULD include the ERC number of the extended ERC in the title (e.g. Non-transferrable [ERC-721](./eip-721.md)). If not in the title, the description MUST include the ERC number of the extended ERC.
+
+### Extensions EIP Header
+
+Once an Extension becomes Final, the extended ERC MUST be edited to include the extension in a new header field called `extensions`.
+
+E.g. ERC-4626 extends ERC-20, therefore ERC-20 must be edited to add: `extensions: 4626`
+
+### Extension Specification
+
+Any breaking changes or non-compliance at the implementation level MUST be fully specified and justified in the `Rationale`. An explicit `ERC-XXX Breaking Changes` sub-heading must be included in the `Specification` section for every extended ERC which has breaking changes. Omission of the Breaking Changes subheading MUST mean there are no breaking changes at the implementation specification level.
+
+All breaking changes MUST be compliant with the properties of an Extension.
+
+EIPs published before [EIP-7539](./eip-7539.md) MAY be edited to comply with some or all of EIP-7539.
```

Comment:
This is a hard no from me.

Reply:
what about just the `extensions` header. Thats the most important and useful edit which doesn't impact content at all

Reply:
This is assuming the `extensions` header is approved pending discussions here and in Eth Magicians